modules/mobile/spec/request/user_request_spec.rb

```diff
@@ -245,6 +248,31 @@
         end
       end
 
+      context 'with a user who does not have access to evss but is using Lighthouse letters service' do
+        before do
+          user = FactoryBot.build(:iam_user, :no_edipi_id)
+          iam_sign_in(user)
+          Flipper.enable(:mobile_lighthouse_letters, user)
+          VCR.use_cassette('payment_information/payment_information') do
+            VCR.use_cassette('user/get_facilities_no_ids', match_requests_on: %i[method uri]) do
+              get '/mobile/v0/user', headers: iam_headers
+            end
+          end
+        end
+
+        it 'does not include edipi services (claims, direct deposit, letters, military history)' do
```

Comment:
When the flag is on and we're using LH for letters, we don't need to validate anything but FE still needs to see it as an authorized service since they won't be making any FE changes for this new upstream service.